opentelemetry-api/src/opentelemetry/context/__init__.py

```diff
@@ -120,9 +104,7 @@ def set_current(context: Context) -> None:
     Args:
         context: The context to use as current.
     """
-    if _CONTEXT_RUNTIME is None:
-        get_current()
-    _CONTEXT_RUNTIME.apply(context.snapshot())
+    _CONTEXT_RUNTIME.set_current(Context(context.snapshot()))
```

Comment:
The spec [requires](https://github.com/open-telemetry/opentelemetry-specification/pull/424/files#diff-0302130d728b6306eda7a69cd92c6350R100) a handle to be returned here.

Reply:
Updated to return the old context, we may need to implement an opaque handle type here, but would rather not block the PR on it